changelog/62477.fixed.md

```diff
@@ -0,0 +1 @@
+Issue #62477: Check NamedLoadedContext mapping is not None, before attempting to create a dictory with it
```

Comment:
```suggestion
Check NamedLoadedContext mapping is not None, before attempting to create a dictory with it
```
The issue number gets appended once rendered.

Reply:
Thanks for that, looked at one fixed in changelog and copied the same format.
Wondering if we updated the doc's to reflect that the 'Issue #123456: ' will get added after ?